flow/project.py

```diff
@@ -415,13 +512,14 @@ class _FlowCondition(object):
     def __init__(self, callback):
         self._callback = callback
 
-    def __call__(self, job):
+    def __call__(self, *jobs):
```

Comment:
This is internal now, so do we need to use `*jobs` here or can we just use the list?

Reply:
A list will be fine.
I'll change this behaviour